agents/node/vcxagent-core/package.json

```diff
@@ -33,8 +33,8 @@
     "demo:faber:pg": "node demo/faber.js --postgresql",
     "demo:alice:sign": "node demo/alice-signature.js",
     "demo:faber:verify": "node demo/faber-verify-signature.js",
-    "test:legacy:integration": "node demo/integration-test.js",
-    "test:legacy:integration:revocation": "REVOCATION=true node demo/integration-test.js",
+    "test:threaded:integration": "node demo/integration-test.js",
```

Comment:
Let me take this opportunity to say I think something like `demo:e2e`, `demo:full` is more accurate and understandable

Reply:
good point, I've renamed to plainly `demo`